### DISCUSSION TITLE
[Keyball 61]左基盤給電時に右基盤のトラックボールが反応しない

### DISCUSSION THREAD

**Discussion by hrtpears** [Yowkees/keyball#568] [Q&A]
お世話になっております。
以下の症状に悩んでいるのですが、基盤のどこを見直せばよいのかが分かりません。
ご教示お願い致します。

【症状】
・左右のPromicroに同じファームウェア、キーマップを書き込み
・トラックボール側のPromicro給電時は問題無く動作
・トラックボール無い側のPromicro給電時にトラックボールが反応しない

![HIDARI_20240427_1](https://github.com/Yowkees/keyball/assets/166685050/f6600275-4cc8-4790-a5ba-ad0f182fad08)

**Comment by Ryo624** [comment]
TRSコネクタのはんだ付けをやり直してみるとどうなりますでしょうか。
端子にしっかりはんだが馴染むようにはんだ付けを行ってみてください。

また、TRSケーブルをほかにお持ちでしたら別のものに変えてみたときの通信の状況も確かめていただけますと幸いです。

**Comment by hrtpears** [comment]
ご回答ありがとうございます。
TRSコネクタのはんだ付けをやり直したところ動作するようになりました。
無事に解決することができて安心しました。

**Comment by Ryo624** [comment]
無事に解決していただきましてありがとうございます。
末永くご愛用いただけますと幸いです。
